### ISSUE TITLE
Build Training Pipeline

### ISSUE THREAD

**Issue by arabind-meher** [arabind-meher/Facial-Emotion-Recognition-with-CNN#4]
Build training loop with optimizer and loss

Add checkpoint saving
